docs/AboutUs.adoc

```diff
@@ -4,53 +4,44 @@ ifdef::env-github,env-browser[:outfilesuffix: .adoc]
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-The dummy content given below serves as a placeholder to be used by future forks of the project. +
+AddressBook - Level 4 was developed by the https://github.com/CS2103AUG2017-T16-B2/main[T16-B2] team. +
```

Comment:
Perhaps the team can come out with another name aside from `Addressbook - Level 4`?